sdk/sdk/Models/dao/OBAModelDAOUserPreferencesImpl.m

```diff
@@ -243,8 +235,6 @@ - (void) writeSetRegionAutomatically:(BOOL)setRegionAutomatically {
     NSUserDefaults * user = [NSUserDefaults standardUserDefaults];
     NSNumber * v = @(setRegionAutomatically);
     [user setObject:v forKey:kSetRegionAutomaticallyKey];
-
-    [[GAI sharedInstance].defaultTracker set:[GAIFields customDimensionForIndex:2] value:(setRegionAutomatically ? @"YES" : @"NO")];
```

Comment:
@beliyd up to @aaronbrethorst but I would be interested in seeing/learning more of those proper abstractions, etc.